src/layers/generic.cpp

```diff
@@ -485,7 +512,7 @@ namespace marian {
     //        - if it is required to be in a different range, the embeddings can still learn that, but more slowly
 
     auto batchEmbeddings = apply(subBatch->data(), {dimWidth, dimBatch, dimEmb});
-#if 0
+#if 1
```

Comment:
If this code is now active, we should get rid of the macro

Reply:
With this I wanted to make sure that the default code did not run the crossMaskWithInlineFixSourceSuppressed() https://github.com/pedrodiascoelho/marian-dev/blob/4c7a1ad3d857ef144ec5e3ff0df6c36161e2ee77/src/data/corpus_base.cpp#L448 both because this function it is said to be experimental, and also because this was trowing a lot of warnings into the log even though it was not actually used. Maintend the #if 0 and switched what is in the #if and in the #else which I think that shows better my intention when I changed this.